micro-ui/web/micro-ui-internals/packages/modules/workbench/src/components/DigitJSONForm.js

```
@@ -175,24 +126,71 @@ function ArrayFieldTemplate(props) {
+        <div key={fieldName} style={{ marginBottom: "1rem" }}>
+          {/* Actual Input Field */}
+          <div className="field-wrapper object-wrapper" id={`${idSchema["$id"]}_${fieldName}`}>
+            {element.content}
+          </div>
+      
+          {/* MDMS and Localization Codes */}
+          <div className="code-details" style={{ padding: "0.5rem 1.5rem", backgroundColor: "#f9f9f9", border: "1px solid #e0e0e0", borderRadius: "4px" }}>
+            <div className="code-row" style={{ display: "flex", alignItems: "flex-start", marginBottom: "0.75rem" }}>
+              <label className="code-key" style={{ flex: "0 0 150px", fontWeight: "bold", color: "#444", textAlign: "left" }}>
+                mdms code:
+              </label>
```

Biome (1.9.4)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)